### PR TITLE
Using Bootstrap: Newsweek Clone

### PR DESCRIPTION
**Pull request by yigitm**
This project's goal is to make a mobile-first, responsive clone of the "Newsweek" web site. It is designed & built with a combination of Bootstrap 4 framework and HTML & CSS features.

**- Except for the specific sections/areas; I focused to give priority for using Bootstrap 4 CSS classes.**

**- Two main breakpoint is used for reponsive design: min-width: 768px. & min-width: 992px.**

Thank you so much for your time & inputs :raised_hands:

### CODE REVIEW

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# Status: Changes required 🛑

## Hello @yigitm 👋, you have done an incredible job making this project. It's one of the best projects I have seen so far 💯. Kudos for less relying on CSS 👌!

### However, there are a couple of changes that you need to work before we can merge.

-  The `search` in the navbar should not be a link. It should be an `input` element.

- You missed the `featured slideshows(us, world, tech & science, ...)` section after `editor's picks`.

#### Clone

image

#### Original

image

image

image

image


- In tablet mode, the social icons in `footer` are overflowing which is causing an extra space to appear to the right side of the clone.

#### The extra space -
Annotation 2020-06-28 165430

#### The issue causing it - 
Annotation 2020-06-28 165119

- I have left one comment on your CSS code below. Be sure to check that.

File: style.css [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+.color-global-blue {
+  color: #3581f4;
+}
+
+.color-global-red {
+  color: #f72210;
+}
+
+.custom-logo {
+  width: 25%;
```
Comment (tirthajyoti-ghosh) [inline style.css]: There is a Bootstrap class for `width: 25%;` property. It is `w-25`. By removing this line and using the Bootstrap counterpart, you will further decrease your CSS code. Be sure to do that.

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
*I have noticed that you have added stickler config files but have not enabled stickler. Be sure to add stickler before your next code review. Even better, you can setup GHA. [Here](https://github.com/microverseinc/linters-config/tree/master/html-css) are the steps.*

Apply these changes and request another code review.

Happy coding 👨‍💻

**Review by elmejdki — CHANGES_REQUESTED** [review]
# Status: require changes :recycle: 
## Great job so far @yigitm :clap: you have made a great :tophat:  clone but in order to get it approved you need to make some minor changes :hammer: :wrench: :toolbox:.

- [x] Fix the header on mobile views it's overlapping:
image

- [x] you have extras space above the culture & travel and also more stories on the desktop view, please remove it:
image

- [x] on the opinion section you have cloned also some ads please remove them:
image
image
also this:
image
### :warning:  I recommend that you install ad-block as an extension for your browser.

- [x] please fix the subscribe section on mobile view, it's out of it' container:
image

- [x] also the form inside the opinion section:
image

- [x] add some margin around the newsletter section:
image

### Please make all the required changes before you submit another code review.
# Happy Coding :tophat:

**Review by elmejdki — APPROVED** [review]
# status Approved :heavy_check_mark: 
## Great job guys, thank you for making the required changes :clap: 

now you can merge to master your project is approved.

## Happy Coding

**Pull request merged**